### DISCUSSION TITLE
Which event to hook to modify footer links?

### DISCUSSION THREAD

**Discussion by lanmaster53** [squidfunk/mkdocs-material#4485] [Q&A]
I built an extension that uses the `on_env` event to sort the TOC by page name. Apparently, this event is the first place I can access the nav object after the page names have been created. Anyway, no matter what I do to the TOC, the footer links always show the prev/next links as if the pages were never sorted. Where are the footer links established? Do they not use the nav object? If not, where and from what are they determined? Thanks!

**Comment by squidfunk** [comment]
This is a general MkDocs question, so better ask it on MkDocs' discussion board.
